apis/fluentbit/v1alpha2/plugins/filter/lua_types.go

```diff
@@ -45,7 +48,28 @@ func (l *Lua) Params(_ plugins.SecretLoader) (*params.KVs, error) {
 	if err != nil {
 		return kvs, err
 	}
-	kvs.Insert("script", "/fluent-bit/config/"+l.Script.Key)
+
```

Comment:
Thanks for the PR @chrono2002 , looks like there're indent issues here.
You may need to run :
```shell
make generate
make manifests
make fmt
make vet
make docs-update
```

Reply:
![image](https://github.com/fluent/fluent-operator/assets/18525465/0a6d524b-d050-4255-95ef-863843de09f0)